src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Build.ps1

```diff
@@ -1228,6 +1231,12 @@ class Build: ADOSVTBase
                                 $formattedBroaderGrpTable = ($formattedGroupsData | Out-String)
                                 $controlResult.AddMessage("`nList of groups : `n$formattedBroaderGrpTable");
                                 $controlResult.AdditionalInfo += "List of excessive permissions on which contributors have access:  $($groupsWithExcessivePermissionsList.Group).";
```

Comment:
We do not need Descriptor, PermissionSetToken, PermissionSetId info in additional info